aviary/interface/reports.py

```diff
@@ -42,3 +52,118 @@ def subsystem_report(prob, **kwargs):
 
     for subsystem in core_subsystems.values():
         subsystem.report(prob, reports_folder, **kwargs)
+
+
+def mission_report(prob, **kwargs):
+    """
+    Creates a basic mission summary report that is place in the "reports" folder
+
+    Parameters
+    ----------
+    prob : AviaryProblem
+        The AviaryProblem used to generate this report
+    """
+    def _get_phase_value(traj, phase, var_name, units, indices=None):
+        try:
+            vals = prob.get_val(f"{traj}.{phase}.timeseries.{var_name}",
+                                units=units,
+                                indices=indices,)
+
+        except KeyError:
+            try:
+                vals = prob.get_val(f"{traj}.{phase}.states:{var_name}",
+                                    units=units,
+                                    indices=indices,)
+            except KeyError:
+                try:
+                    vals = prob.get_val(f"{traj}.{phase}.timeseries.states:{var_name}",
+                                        units=units,
+                                        indices=indices,)
+                except KeyError:
+                    try:
+                        vals = prob.get_val(f"{traj}.{phase}.{var_name}",
+                                            units=units,
+                                            indices=indices,)
+                    # 2DOF breguet range cruise uses time integration to track mass
+                    except TypeError:
+                        vals = prob.get_val(f"{traj}.{phase}.timeseries.time",
+                                            units=units,
+                                            indices=indices,)
+                    except KeyError:
+                        vals = None
```

Comment:
Do all of these try/except blocks strictly need to be here? I'm thinking that most of the values we care about from the mission should be added to the timeseries (so they'd be accessible in the first try block).

Reply:
Good point. I did some tests and some of these aren't required. I've removed the unneeded try/except blocks